server/services/utils/formValidation.js

```diff
@@ -134,13 +134,20 @@ function validate({formResponse, pageConfig, formType = 'standard', bespokeCondi
     return joiErrors.error.details.reduce((errors, error) => {
         // joi returns map to error in path field
         const fieldConfig = fieldsConfig.find(field => getFieldName(field) === error.path[0]);
-        const errorMessage = procedure.getErrorMessage(fieldConfig, error.path) || error.message;
+        const errorMessage = getErrorMessage(fieldConfig, error, procedure.getErrorMessage);
 
         const errorObject = error.path.reduceRight((errorObj, key) => ({[key]: errorObj}), errorMessage);
         return mergeWithRight(errors, errorObject);
     }, {});
 }
 
+function getErrorMessage(fieldConfig, error, errorMethod) {
+    if (error.type === 'date.min') {
+        return 'Enter a date that is in the future';
```

Comment:
I can't really tell, but what happens if you enter today's date?  Is that allowed?

Reply:
It does reject today, which I assume was specified when it was originally
implemented. Will run past Jim